flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
this is disappointing.

Reply:
It's because on ARM, Qt is only built with support for GLES, not the complete OpenGL. So either that would need to be changed, or, probably easier and better, Syntalos would need to be ported to GLES.
The bulk of direct GL calls is in https://github.com/bothlab/syntalos/blob/master/modules/canvas/imageviewwidget.cpp, but it's still annoying porting work :-/